lib/MusicBrainz/Server/Form/Role/EditNote.pm

```diff
@@ -18,9 +19,13 @@ has_field 'make_votable' => (
 after validate => sub {
     my $self = shift;
 
-    if ($self->requires_edit_note && (!defined $self->field('edit_note')->value || $self->field('edit_note')->value eq '')) {
+    if ($self->requires_edit_note && (!defined $self->field('edit_note')->value)) {
         $self->field('edit_note')->add_error(l('You must provide an edit note'));
     }
+
+    if ($self->requires_edit_note && (!is_valid_edit_note($self->field('edit_note')->value))) {
+        $self->field('edit_note')->add_error(l('Your edit note seems to have no actual content. Please provide a note that will be helpful to your fellow editors!'));
```

Comment:
It would probably be helpful to link it to [`/doc/How_to_Write_Edit_Notes`](https://musicbrainz.org/doc/How_to_Write_Edit_Notes)

Reply:
This doesn't seem to work on error texts at the moment (I got `You must provide <a href="/doc/How_to_Write_Edit_Notes">an edit note</a>`) but in any case it seems like it would be relevant always, not just if an error appears. So I added [MBS-10373](https://tickets.metabrainz.org/browse/MBS-10373) for that and added it to the general edit note text.